src/corg/development-tools-and-process/git-usage.md

```diff
@@ -681,25 +683,54 @@ git reset --soft HEAD~1
+	--	Optional separator between the options and the instructions which may start with a dash.
+
+gitfa -?
+Usage: gitfa [-q] [-h|-?] <[--] Git instructions>
+Recursively executes the given Git command on the current, its direct children Git repositories and browse the child Maven modules of $PARENT_MODULES directories.
+
+	-h,-?	Show this help message and exit.
+	-q	Quiet mode (default is verbose).
+	--	Optional separator between the options and the instructions which may start with a dash.
+	PARENT_MODULES Environment variable. Defaults to 'addons addons-core' if not set.
+	LIST_<parent module> Environment variable that can be set to restrict the children modules to a fixed list. For instance: LIST_ADDONS_CORE="nuxeo-core-storage-marklogic" or LIST_ADDONS="nuxeo-shell nuxeo-quota".
+
+shr -?
+Usage: shr [-a] [-q] [-h|-?] <[--] Shell instructions>
+Recursively executes the given Shell command on the current and its direct children Git repositories.
+
+	-h,-?	Show this help message and exit.
+	-a	The recurse will continue on the child Maven modules of $PARENT_MODULES directories.
+	-q	Quiet mode (default is verbose).
+	--	Optional separator between the options and the instructions which may start with a dash.
+	PARENT_MODULES Environment variable. Defaults to 'addons addons-core' if not set.
+	LIST_<parent module> Environment variable that can be set to restrict the children modules to a fixed list. For instance: LIST_ADDONS_CORE="nuxeo-core-storage-marklogic" or LIST_ADDONS="nuxeo-shell nuxeo-quota".
 ```
 
 ### Main Rules
 
-*   Strictly follow Nuxeo naming policy for branches and tags:
-
-    *   Create dedicated branches named "fix-NXP-1234-some-expected-behavior" or "feature-NXP-1234-some-new-feature" while working.
-    *   When your work is achieved, rebase then merge the branch (see [Isolated testing with ondemand-testandpush jobs]({{page page='isolated-testing'}})). Optionally delete it.
-*   Use the same name for a local branch than its corresponding remote branch.
-*   Always reference JIRA issues in the commit comments.
-*   Commit as a valid author (i.e. "Firstname Lastname <address@somewhere.com>" with "address@somewhere.com" being configured in your GitHub account).
+* Avoid to rewrite Git history on development and maintenance branches.
+* Strictly follow Nuxeo naming policy for branches and tags:
+    * Create dedicated branches named "fix-NXP-1234-some-expected-behavior" or "feature-NXP-1234-some-new-feature" while working.
+    * When your work is achieved, rebase then merge the branch (see [Isolated testing with ondemand-testandpush jobs]({{page page='isolated-testing'}})). Optionally delete it.
+* Use the same name for a local branch than its corresponding remote branch.
+* Always reference JIRA issues in the commit comments.
+* Commit as a valid author (i.e. "Firstname Lastname <address@somewhere.com>" with "address@somewhere.com" being configured in your GitHub account).
```

Comment:
`example.com` is an official domain name explicitly designed for documentation and pseudo code.